compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/coneDiagnosticToFirDiagnostic.kt

```diff
@@ -101,20 +100,25 @@ private fun mapUnsafeCallError(
     val receiverExpression = candidate.callInfo.explicitReceiver
     val singleArgument = candidate.callInfo.argumentList.arguments.singleOrNull()
     if (receiverExpression != null && singleArgument != null &&
-        source.elementType == KtNodeTypes.OPERATION_REFERENCE &&
+        (source.elementType == KtNodeTypes.OPERATION_REFERENCE || source.elementType == KtNodeTypes.BINARY_EXPRESSION) &&
         (candidateFunction?.isOperator == true || candidateFunction?.isInfix == true)
     ) {
-        val operationToken = source.getChild(KtTokens.IDENTIFIER)
-        if (candidateFunction.isInfix && operationToken?.elementType == KtTokens.IDENTIFIER) {
-            return FirErrors.UNSAFE_INFIX_CALL.on(
+        // For augmented assignment operations (e.g., `a += b`), the source is the entire binary expression (BINARY_EXPRESSION).
+        // TODO: Use operator as callee reference source (see FirExpressionsResolveTransformer.transformAssignmentOperatorStatement)
```

Comment:
So after callee reference source is set to the operator in  FirExpressionsResolveTransformer.transformAssignmentOperatorStatement, the change here can be reverted, correct?

Reply:
Yes, that's correct. I'll clarify the comment.